### PR TITLE
[Fixes Travis Failing] Removed hammer options help from factory helper docstrings

### PR DESCRIPTION
**Pull request by jyejare**
This fixes :
- [x] Removed hammer options help from factory helper docstrings
- [x] The Travis failing for using `literal` in docstrings issue.

### CODE REVIEW

File: robottelo/cli/factory.py [hunk 1]
```diff
@@ -236,73 +214,19 @@ def make_architecture(options=None):
...
-        --repositories REPOSITORY_NAMES Comma separated list of values.
-        --repository-ids REPOSITORY_IDS List of repository ids
-                                        Comma separated list of values.
-        -h, --help                    print help
-
-    If credentials is None default credentials present on
-    robottelo.properties will be used.
+    """Helper function to create CV with credentials
 
+    If credentials is None default credentials present on robottelo.properties will be used.
```
Comment (swadeley) [inline robottelo/cli/factory.py:229]: If credentials is None, the default credentials in robottelo.properties will be used.

File: robottelo/cli/factory.py [hunk 2]
```diff
@@ -438,40 +311,11 @@ def make_content_view_filter_rule(options=None):
 
 @cacheable
 def make_discoveryrule(options=None):
-    """
-    Usage::
+    """Creates a Discovery Rule
+
+    :param options: Check options using `hammer discovery_rule create --help` on satellite.
```
Comment (swadeley) [inline robottelo/cli/factory.py:316]:
```suggestion
    :param options: Check options using `hammer discovery-rule create --help` on satellite.
```

File: robottelo/cli/factory.py [hunk 3]
```diff
@@ -1250,162 +772,11 @@ def make_sync_plan(options=None):
 
 @cacheable
 def make_host(options=None):
-    """
-    Usage::
+    """Creates/Provisions a Host
```
Comment (swadeley) [inline robottelo/cli/factory.py:775]:
```suggestion
    """Creates a Host
```

File: robottelo/cli/factory.py [hunk 4]
```diff
@@ -2175,36 +1237,11 @@ def make_org_with_credentials(options=None, credentials=None):
 
 @cacheable
 def make_realm(options=None):
-    """
-    Usage::
+    """Creates a REALM
```
Comment (swadeley) [inline robottelo/cli/factory.py:1240]: Does realm have to be capitalised?

File: robottelo/cli/factory.py [hunk 5]
```diff
@@ -2317,44 +1296,11 @@ def make_report_template(options=None):
 
 @cacheable
 def make_os(options=None):
-    """
-    Usage::
+    """Creates an Opertaing System
```
Comment (swadeley) [inline robottelo/cli/factory.py:1299]:
```suggestion
    """Creates an Operating System
```

File: robottelo/cli/factory.py [hunk 6]
```diff
@@ -2380,33 +1326,11 @@ def make_os(options=None):
 
 @cacheable
 def make_scapcontent(options=None):
-    """
-    Usage::
+    """Creates a Scap Content
```
Comment (swadeley) [inline robottelo/cli/factory.py:1329]:
```suggestion
    """Creates Scap Content
```

File: robottelo/cli/factory.py [hunk 7]
```diff
@@ -2833,86 +1574,11 @@ def make_template(options=None):
 @cacheable
 def make_template_input(options=None):
     """
-    Usage::
+    Creates a Template Imput
```
Comment (swadeley) [inline robottelo/cli/factory.py:1577]:
```suggestion
    Creates template input
```

File: robottelo/cli/factory.py [hunk 8]
```diff
@@ -3091,16 +1638,7 @@ def make_virt_who_config(options=None):
...
-    Adds subscription to activation key.
-
-    Args::
-
-        organization-id - ID of organization
-        activationkey-id - ID of activation key
-        subscription - subscription name
-
-    """
+    """Helper function that adds subscription to activation key"""
```
Comment (swadeley) [inline robottelo/cli/factory.py:1641]:
```suggestion
    """Helper function that adds subscription to an activation key"""
```

File: robottelo/cli/factory.py (hunk 4 above)
Comment (jyejare) [reply robottelo/cli/factory.py:1240]: No, it not.

**Comment by jyejare** [comment]
@swadeley Fixed comments! Thanks for the review!

@SatelliteQE/quality-engineers Needs a quick merge to fix the travis.

**Comment by lpramuk** [comment]
This PR breaks pytest collection !!!
```
__________________________________ ERROR collecting tests/foreman/cli/test_host.py ____________________________________
ImportError while importing test module '/home/lpramuk/git/SatelliteQE/robottelo/tests/foreman/cli/test_host.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/foreman/cli/test_host.py:54: in <module>
    from robottelo.cli.factory import make_smart_variable
E   ImportError: cannot import name 'make_smart_variable' from 'robottelo.cli.factory' (/home/lpramuk/git/SatelliteQE/robottelo/robottelo/cli/factory.py)
________________________________ ERROR collecting tests/foreman/cli/test_puppetclass.py ________________________________
ImportError while importing test module '/home/lpramuk/git/SatelliteQE/robottelo/tests/foreman/cli/test_puppetclass.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/foreman/cli/test_puppetclass.py:20: in <module>
    from robottelo.cli.factory import make_smart_variable
E   ImportError: cannot import name 'make_smart_variable' from 'robottelo.cli.factory' (/home/lpramuk/git/SatelliteQE/robottelo/robottelo/cli/factory.py)
_________________________________ ERROR collecting tests/foreman/cli/test_variables.py _________________________________
ImportError while importing test module '/home/lpramuk/git/SatelliteQE/robottelo/tests/foreman/cli/test_variables.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/foreman/cli/test_variables.py:27: in <module>
    from robottelo.cli.factory import make_smart_variable
E   ImportError: cannot import name 'make_smart_variable' from 'robottelo.cli.factory' (/home/lpramuk/git/SatelliteQE/robottelo/robottelo/cli/factory.py)
--------------- generated xml file: /home/lpramuk/git/SatelliteQE/robottelo/tier1-sequential-results.xml ---------------
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 3 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Pull request merged**